docs/index.md

```diff
@@ -525,17 +526,28 @@ sometimes prove many statements automatically.
 > Enter, for this new step, the statement
 > `|- ( 3 + 1 ) = ( ( 2 + 1 ) + 1 )`
 > and press Enter (Return).
-> Now, while no steps are selected, click on the
-> icon <img width="16" height="16" src="hub.svg" alt="Unify"> (unify).
-> Since there
-> was no specific step selected, it will try to justify all steps.
 
-Metamath-lamp will succeed in finding a justification for our new step,
+By default Metamath-lamp automatically runs a simple unification process
+to justify any steps that aren't yet justified.
+Though this process,
```

Comment:
typo `Though` -> `Through`

Reply:
Thanks for reporting this! I'll fix it in a separate pull request.